scripts/app/gamAppAds.js

```diff
@@ -146,22 +209,28 @@ var ksfDfp = {};
 
             googletag.pubads().addEventListener("slotRenderEnded", function(event) {
                 if (typeof event !== 'undefined' && event.size != null) {
+                    // move slots that are to be repositioned in the document
+                    if (!ksfDfp.mover.done) {
+                        setTimeout(() => {
+                            ksfDfp.bannerMover();
+                        }, 200);
+                    }
                     var contentUnitDiv;
                     var headerToShow;
                     contentUnitDiv = event.slot.getSlotElementId();
 
                     // show ad headline fitting to the ad format. Out of page slots should have a headline that allows viewers to close the ad
                     // only show ad hedline for ads that are actually supposed to render
-                    contentUnitDiv = event.slot.getSlotElementId(); // this line is probably not needed?
+                    //                    contentUnitDiv = event.slot.getSlotElementId(); // this line is probably not needed?
```

Comment:
Rather than having commented code, can we just remove this line?

Reply:
Done